README.md

```diff
@@ -1 +1,37 @@
-# hw01
\ No newline at end of file
+# hw01
+
+# Self Introduction
+Hello. My name is __Soomin Kim__. I am a MA student of _Committee on International Relations_. I was born and raised in South Korea for 23 years and came to the United States to continue my study. I have one younger sister, who is also studying in the Unites States. We love and take care of each other for we are the only family members outside of South Korea. 
+
+# Research Interest
+I have three main research interests.
+
+1. **Nuclear Security**
+
+    Coming from South Korea, I was always exposed to North Korea's nuclear weapon development. Whenever the nuclear missile testing was done, the whole world was reacting to it. I was curious, how can we predict North Korea's behavior in terms of nuclear weapons?, what is the real motivation for them to do so. 
+
+2. **Political Economy**
+
+    World now imposes economic sanctions against North Korea as a warning sign. Does that work? Will North Korea come out to the table and start the talk? If the economic sanctions are not effective, why is that? I think a newly emerging capitalist group, *Donju* dilutes the effectiveness. 
+
+1. **Regional Interest**
+
+    + Northeast Asia: China, North Korea, South Korea, Japan
+    + Latin America: Cuba
+
+# Hobbies
+## **Youtube**
+
+When I first came to the United States, my grandparents were very worried about me. Also, my parents wanted to see how my days are going. So, I decided to film myself and post it on Youtube for my family.
+
+[Lemon's Diary](https://www.youtube.com/channel/UCtnNSkVJe3K9GxoXFMyEFIg?view_as=subscriber) is my channel name. Click on the channel name and the youtube page will open.
+
+## **Playing with my puppy, Geni**
+
+My lovely puppy, __Geni__, is a 3 year-old poodle. He is very friendly, lively and lovely. When I was back home in Korea, I loved to have a short walk with him everyday. It was a happy distraction from all the works I needed to do. 
+
+![Picture of Geni](Genipicture.jpeg)
```

Comment:
For some reason (I don't know) the picture appears at the beginning of the bio, instead of here. I had problems with adding the image, and the way I tackled that was adding the picture doc to the repository.